File: Java Barcode Scanner ME/app/src/main/java/de/dimedis/mobileentry/backend/response/AccessConfig.java
package de.dimedis.mobileentry.backend.response;

import com.google.gson.annotations.SerializedName;

public class AccessConfig {
    public static final String ARBITRARY_DATA = "Arbitrary data";


    @SerializedName(ARBITRARY_DATA)
    private String arbitrary_data;

    public String getArbitraryData() {
        return arbitrary_data;
    }

}

File: Kotlin Migration/NTMobileEntry/app/src/main/java/de/dimedis/mobileentry/backend/response/AccessConfig.kt
package de.dimedis.mobileentry.backend.response

import com.google.gson.annotations.SerializedName

class AccessConfig {
    @SerializedName(ARBITRARY_DATA)
    private val arbitrary_data: String? = null
    val arbitraryData: String?
        get() = arbitrary_data

    companion object {
        const val ARBITRARY_DATA = "Arbitrary data"
    }
}